api/types/constants.go

```diff
@@ -127,6 +127,9 @@ const (
 	// KindNode is node resource
 	KindNode = "node"
 
+	// KindOpenSSHNode is a Teleport node.
```

Comment:
```suggestion
	// KindTeleportNode is a Teleport node.
```

Despite the copy paste error in the doc, this is still not clear to me. `KindNode` is also a Teleport node, no?

Reply:
Done